tools/failures.py

```diff
@@ -11,14 +11,18 @@
 
 import seta
 
-def getRawData():
-    url = "http://alertmanager.allizom.org/data/seta/"
+def getRawData(startDate, endDate):
```

Comment:
I know we're not super consistent in Ouija, but these should probably be start_date and end_date.